### ISSUE TITLE
[LC] Learning Center category fields config and authoring experience

### ISSUE THREAD

**Issue by oksana-c** [department-of-veterans-affairs/va.gov-cms#2872]
## Acceptance Criteria

- [ ] the following two fields exist on all LC content types

**updated ACs 09/21/20**

field machine name | label | other properties
-- | -- | -- 
 ~`field_lc_primary_category`~  `field_primary_category` | Select a primary Learning Center category for your article |  entity reference to ~Topics~ "Learning Center Categories" taxonomy; cardinality - single; **required** ; form widget - select list
~`field_lc_other_categories`~  `field_primary_category` | Other categories where people may look for this article | entity reference to ~Topics~ "Learning Center Categories" taxonomy; cardinality - unlimited; **optional** ; form widget - entity browser ~Topics~ "Learning Center Categories" (new - needs to be configured). ~One term cannot be referenced twice in this field (OOTB entity ref field allows duplicates).~ > duplicate issue will be handled in a separate story once #2941 is merged.
- [ ] form display settings + node display settings
    * both fields are grouped in "Learning Center categories" field group and displayed under Metatag field group on all LC content types: Q&A, FAQ - multiple Q&As, Checklist, Step-by-step, Media list- Images, Media list - Videos

**Comment by oksana-c** [comment]
Updated ACs for both fields: instead of Topics vocab, both fields should reference terms from "Learning Center Categories"  taxonomy created in #2924
